src/content/basics/multiaz/index.md

```diff
@@ -4,7 +4,17 @@ description: Using multiple availability zones both for worker and for master no
 weight: 100
 type: page
 categories: ["basics"]
-last_review_date: 2020-04-09
```

Comment:
Why do you delete the last_review_date?

Reply:
They seem to have disappeared from other docs and I'm not sure how usable this field will be for us.
Happy to hear thoughts about the necessity of the field.

Reply:
We only had it on very few documents, since we haven't introduced the review process yet. I would leave it where we have it.